index.js

```diff
@@ -2,7 +2,8 @@
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 1", "color: red") 
-
+var counter = 1; 
```

Comment:
`var` works here, but get in the habit of using `let` instead (`var` has some tricky behavior, so you won't see it used as much in modern Javascript).